src/components/CreateTodo.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="createTodo card">
+    <div class="createTodo__inputBox">
+      <label for="createTodo__inputBox_titleLabel">Title</label>
+      <input
+        type="text"
+        class="createTodo__inputBox_titleInput"
+        :value="todoTitle"
+        @input="$emit('title-input', $event.target.value)"
+      />
+      <span v-show="isTitleError">Can't add empty title.</span>
+
+      <label for="createTodo__inputBox_descLabel">Description</label>
+      <textarea
+        class="createTodo__inputBox_descInput"
+        :value="todoDesc"
+        @input="$emit('desc-input', $event.target.value)"
+      />
+      <span v-show="isDescError">Can't add empty description.</span>
+    </div>
+
+    <div class="createTodo__button">
+      <button class="createTodo__button_add btn" @click="$emit('add')">
+        Add
+      </button>
+      <div class="createTodo__button_cancel btn" @click="$emit('cancel')">
+        <DeleteIcon />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import DeleteIcon from "@/components/icons/DeleteIcon.vue"
+
+export default {
+  components: {
+    DeleteIcon,
+  },
+  props: {
+    todoTitle: {
+      type: String,
```

Comment:
Is it required? If not use a default value. In this case the default value will be `null`

Reply:
Do this in other places.